rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -1193,9 +1193,9 @@ object Reduce {
       def size(baseExpr: Expr): M[Par] =
         baseExpr.exprInstance match {
           case EMapBody(ParMap(basePs, _, _, _)) =>
-            Applicative[M].pure[Par](GInt(basePs.size))
+            Applicative[M].pure[Par](GInt(basePs.size.toLong))
```

Comment:
Why do we need this?

Reply:
b/c we migrated to 64-bit integers (longs) as representation and I didn't notice the warnings back then